addon/components/popover-on-element.js

```diff
@@ -1,13 +1,14 @@
 import Ember from 'ember';
 import TooltipAndPopoverComponent from 'ember-tooltips/components/tooltip-and-popover';
-
+import template from 'ember-tooltips/templates/components/popover-on-element';
 
 const { $, run } = Ember;
 
 export default TooltipAndPopoverComponent.extend({
 
   hideDelay: '250',
 
+  template,
```

Comment:
you need to use layout instead of template, template is the block, layout is what yields